media/css/firefox/enterprise-sla.scss

```diff
@@ -0,0 +1,359 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+
+@import '../../protocol/css/includes/lib';
+@import '../../protocol/css/templates/main-with-sidebar';
+@import '../../protocol/css/components/sidebar-menu';
+@import '../../protocol/css/components/article';
+
+$heading-purple: #20123a;
+$body-purple: #42425a;
+$border: 2px solid $color-gray-20;
+
+.mzp-t-firefox {
```

Comment:
Is this something that should be in the design system but isn't? Rather than adding page-specific exceptions, I'd suggest we file a protocol issue for this, and in the meantime default to the regular theme styles that are in the latest Protocol release?